docs/av-stack-sync.1.md

```diff
@@ -0,0 +1,61 @@
+# av-stack-sync 1 "" av-cli "Aviator CLI User Manual"
+
+# NAME
+
+av-stack-sync - Synchronize stacked branches
+
+# SYNOPSIS
+
+`` av stack sync [--all | --current] [--no-push] [--no-fetch] [--prune]
+              [--trunk] [--continue | --abort | --skip] [--parent=<parent>]
+``
+
+# DESCRIPTION
+
+Synchronize stacked branches to be up-to-date with their parent branches.
+
+By default, this command will sync all branches starting at the root of the
+stack and recursively rebasing each branch based on the latest commit from the
+parent branch.
```

Comment:
please also add a line to describing that this does not update references from remote, but pushes the new references to remote.

Reply:
Done.